base_layer/wallet_ffi/src/lib.rs

```diff
@@ -212,26 +212,31 @@ pub struct TariSeedWords(Vec<String>);
 #[derive(Debug, PartialEq)]
 pub struct TariPublicKeys(Vec<TariPublicKey>);
 
+#[repr(C)]
 pub struct TariWallet {
-    wallet: WalletSqlite,
-    runtime: Runtime,
-    shutdown: Shutdown,
+    pub wallet: WalletSqlite,
```

Comment:
Are these needed?

Reply:
Not really, not yet.

Reply:
It doesn't do any harm though, but not much use too because the rest is still only just `typedef`s.